templates/operations.html

```diff
@@ -226,7 +225,7 @@ <h2 class="is-size-7 has-text-weight-normal mb-5 mt-0">Decisions</h2>
                                 <thead>
                                 <tr class="table-header">
                                     <th class="decideColumn">Decide</th>
```

Comment:
With the new UTC timestamps, let's make it easier for the user to see when a step completes. The date/time it now returns can be converted to a locale string, bonus points if it can show the timezone too.

For example, 2021-11-23T17:33:26Z turns into 12:33:26 PM 11-23-2021 EST (for me on the east coast) or something similar